VMBackup/main/logbackup.py

```diff
@@ -0,0 +1,48 @@
+import os
+import re
+import sys
+import subprocess
+import threading
+from workloadPatch.logbackupPatch import logbackup
+from time import sleep
+from datetime import datetime
+
```

Comment:
@shridpant - insert a sample parameter file as comments for regex parsing

Reply:
Done